### ISSUE TITLE
Videos don't load in meme command + End interaction button

### ISSUE THREAD

**Issue by Dhravya** [Dhravya/Spacebot#25]
I noticed that videos simply don't load in the `meme` command. The bigger issue is that I don't really know how to work around this. Any suggestions?
<img width="454" alt="image" src="https://user-images.githubusercontent.com/63950637/145683718-574ae899-5368-4f0d-8a54-d89a89b589f4.png">

Also, an `end interaction` button could be nice that just disables the buttons.

**Comment by MasterBaiter123** [comment]
Its easy tbh. Check if the url of the submission starts with "https://i.redd.it/"

This is how i do in one of my bots-
```py
async for submission in hot:
    if submission.url.startswith("https://i.redd.it/"):
    self.new_memes.append(submission)
```

**Comment by Dhravya** [comment]
Checking is simple, but what about embedding the video? Is it even possible?

**Comment by MasterBaiter123** [comment]
No. At least i haven't successfully found a way. 

You can send the `permalink` of the submission and discord will natively embed it. 
The videos never played on my laptop tho. I had to click on the link to watch them

**Comment by Dhravya** [comment]
Oh, i just discovered this
```py
        url = data[0]["data"]["children"][0]["data"]["secure_media"]["reddit_video"][
            "fallback_url"
        ]
  ```

The above snippet returns the direct .mp4 url of the video, but how to attach it?

**Comment by MasterBaiter123** [comment]
Yeah i can't seem to find a method that adds video links to embed. 

Try setting it as the url during definition of the embed. I'll check and come back

**Comment by Dhravya** [comment]
Yea maybe should just detect the video and make a hyperlink then

**Comment by MasterBaiter123** [comment]
This worked?

**Comment by Dhravya** [comment]
nope didnt work

I don't think embeds can accept video urls lel the video just doesnt load
